examples/triangle/main.rs

```diff
@@ -102,9 +101,10 @@ fn main() {
             }
         }
 
-        graphics.clear(clear_data, gfx::COLOR, &frame);
-        graphics.draw(&batch, &(), &frame).unwrap();
-        graphics.end_frame();
+        renderer.reset();
+        renderer.clear(clear_data, gfx::COLOR, &frame);
+        renderer.draw(&(&mesh, slice, &program, &(), &state), &frame).unwrap();
```

Comment:
Certainly, the error message will not be too helpful. It will just say "Can't find implementation of `Batch` for (your tuple here)". However, adding a special method for it doesn't seem to be a good solution. Perhaps, we could have a named struct instead of the tuple:
```rust
renderer.draw(&gfx::batch::Implicit(&mesh, slice, &program, &(), &state), &frame).unwrap();
```
This will be more wordy but will produce better error messages. Also please note that `RefBatch` has the same problem (with tuple errors), but it would be highly uncomfortable to convert the tuple to another struct there.

All in all, I believe we can add this (the original PR) and decide later, thinking about both `RefBatch` and this calls as parts of the same problem.